src/BatteryStateOfChargeService.h

```diff
@@ -17,6 +17,13 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
 private:
     // Convention is to have an _ after private members
     const double initialStateOfChargePercent_;
-
+    double AvgCurrent;
```

Comment:
Variables should be camelCase, so new words are uppercase except for the first word. As a note, abbreviations should also be lowercase, like `theMvpOfTheNight`.